libs/core/src/lib/upload-collection/upload-collection-simple.directives.ts

```diff
@@ -0,0 +1,53 @@
+import { ContentChildren, Directive, ElementRef, HostBinding, QueryList } from '@angular/core';
+import { ObjectMarkerComponent } from '@fundamental-ngx/core/object-marker';
+
+@Directive({
+    // tslint:disable-next-line: directive-selector
+    selector: '[fd-upload-collection-thumbnail]',
+    host: { class: 'fd-upload-collection__thumbnail' }
+})
+export class UploadCollectionThumbnailDirective {}
+
+@Directive({
+    // tslint:disable-next-line: directive-selector
+    selector: '[fd-upload-collection-title]',
+    host: { class: 'fd-upload-collection__title' }
+})
+export class UploadCollectionTitleDirective {
+    constructor(public elRef: ElementRef) {}
+}
+
+@Directive({
+    // tslint:disable-next-line: directive-selector
+    selector: '[fd-upload-collection-description]',
+    host: { class: 'fd-upload-collection__description' }
+})
+export class UploadCollectionDescriptionDirective {}
+
+@Directive({
+    // tslint:disable-next-line: directive-selector
+    selector: '[fd-upload-collection-text-separator]',
```

Comment:
I think it's better to use 'fd-upload-collection-text-separator' selector, as it's not expected the user to manipulate this element (add class, event, etc.) Or even better have the two options.